package.json

```
@@ -9,7 +9,11 @@
     "tsc": "tsc",
     "lint": "eslint . --ext .ts,.js && tsc --noEmit",
     "build": "rimraf ./dist && tsc",
-    "start": "yarn build && node dist/app.js"
+    "start": "yarn build && node dist/app.js",
+    "docker-start": "docker-compose up -d",
+    "docker-stop": "docker-compose down",
+    "docker-build": "docker-compose build",
+    "docker-start-build": "docker-compose up -d --build"
```

I don't like running docker in detach mode tbh, I love seeing logs, but MongoDB makes too much noise, you can add this to only view logs from the server.

```suggestion
    "docker-start-server-logs": "docker-compose up -d && docker-compose logs -f server"
```

---

I actually prefer it with the logs too so I run it without detach or check the logs in docker desktop